apps/backend/db/migrations/20250820185036_slack-channel-id.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * @param {import('knex').Knex} knex
+ */
+export const up = async (knex) => {
+  // Update automation_action_runs to set "channelId" to the Slack ID
+  await knex.raw(
```

Comment:
I have imported data locally and try to run it. Nothing happened. I think this query is broken